### PR TITLE
Updates docker yarn run to use --production

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates docker yarn run to use --production

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check nothing is broken

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ RUN apt update && \
     curl -fsSL https://deb.nodesource.com/setup_16.x | bash - && \
     apt update && \
     apt install nodejs --yes
-RUN npm install -g yarn
+RUN npm install -g yarn --production
```
Comment (hatched) [inline Dockerfile:11]: This change is a noop. the `--production` flag only does something when it's installing from a package.json. You probably meant to add this to line 15 instead.

**Review by hatched — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (hatched) [reply Dockerfile:11]: Sorry ignore that previous comment, this is still a noop but if you move it to 15 then it won't have the necessary dependencies to actually build the project :)

**Comment by petesfrench** [comment]
@hatched Hi, I moved the production tag to the other line as per your suggestion and played around with the dependencies to get it to build. I think it should work fine now?

**Review by hatched — APPROVED** [review]

**Comment by hatched** [comment]
Thanks for getting this working it's much faster to build.

After some digging the reason these random deps had to be moved up is because CRA runs lint on build.

**Pull request merged**